WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/details/RestoreDetailsStateListItemBuilder.kt

```diff
@@ -0,0 +1,80 @@
+package org.wordpress.android.ui.jetpack.restore.details
+
+import dagger.Reusable
+import org.wordpress.android.R
+import org.wordpress.android.R.string
```

Comment:
Minor (🔍): It seems that this is a wrongly added import. Consider removing it.

PS: Removing this will require you to fix this `restore_details_choose_items_header` at the end of this class.

Reply:
In commit 3681b2b